client.go

```diff
@@ -26,8 +26,10 @@ type Client struct {
 	plugins []ClientPlugin
 }
 
-func (c *Client) AddPlugin(plugin ClientPlugin) {
-	c.plugins = append(c.plugins, plugin)
+func (c *Client) AddPlugin(plugins ...ClientPlugin) {
```

Comment:
Nit: AddPlugins?

Reply:
Great point. I think we want to make this private and move this into the constructor, only. I am going to update #17 and deprecate the old AddPlugin method